content/docs/render-props.md

```diff
@@ -299,27 +299,27 @@ class MouseTracker extends React.Component {
 }
 ```
 
-In this example, each time `<MouseTracker>` renders, it generates a new function as the value of the `<Mouse render>` prop, thus negating the effect of `<Mouse>` extending `React.PureComponent` in the first place!
+Dans cet exemple, à chaque fois que `<MouseTracker>` est rendu, une nouvelle fonction est générée comme valeur de la propriété `<Mouse render>`, ce qui annule l'effet de `<Mouse>` qui étend `React.PureComponent` !
 
-To get around this problem, you can sometimes define the prop as an instance method, like so:
+Pour éviter ce problème; vous pouvez parfois définir la propriété comme une méthode d'instance, de la façon suivante :
 
 ```js
 class MouseTracker extends React.Component {
-  // Defined as an instance method, `this.renderTheCat` always
-  // refers to *same* function when we use it in render
+  // Définie comme une méthode d'instance, `this.renderTheCat` se réfèrera
+  // toujours à la *même* fonction quand nous l'utiliserons dans le rendu
   renderTheCat(mouse) {
     return <Cat mouse={mouse} />;
   }
 
   render() {
     return (
       <div>
-        <h1>Move the mouse around!</h1>
+        <h1>Déplacez votre souris sur l'écran !</h1>
         <Mouse render={this.renderTheCat} />
       </div>
     );
   }
 }
 ```
 
-In cases where you cannot define the prop statically (e.g. because you need to close over the component's props and/or state) `<Mouse>` should extend `React.Component` instead.
+Dans les cas où vous ne pourrez pas définir la propriété de manière statique (e.g. parce que vous devez fermer la propriété et/ou l'état local) `<Mouse>` devrait plutôt étendre `React.Component`.
```

Comment:
```suggestion
Dans les cas où vous ne pourrez pas définir la prop de manière statique (ex. parce qu’elle utilise des valeurs locales à `render`), `<Mouse>` devrait plutôt étendre `React.Component`.
```

Les raisons invoquées sont en fait difficiles à capter, puisqu'on est dans une classe et non une fonction composant : les props et l'état local sont donc accessibles via `this`, même si on y perd la garantie de l'identité du rendu. C'est plus pratique de parler des identifiants locaux du `render` pour illustrer la question.